tests/e2e/data/help-contents.ts

```diff
@@ -23,6 +23,8 @@ Commands:
   taq init [projectDir]                  Initialize a new project
   taq scaffold [scaffoldUrl] [scaffoldP  Generate a new project using a pre
   rojectDir]                             -configured scaffold
+  taq optin                              Opt in for Google Analytics
```

Comment:
For discussion: Should the taskname be `optin`/`optout`, or `opt-in`/`opt-out`?

Reply:
The former is easier to type so I'd prefer that one.

Reply:
fixed